tools/schemacode/pyproject.toml

```
@@ -29,3 +29,8 @@ multi_line_output = 3
 markers = [
   "validate_schema: tests that validate the schema itself",
 ]
+
+[coverage:run]
+parallel = True
+omit =
+    bidsschematools/tests/*py
```

TOML syntax.

```suggestion
parallel = true
omit = ["*/tests/*"]
```

Also, I think we need to update the test dependencies in setup.cfg to use `coverage[toml]` to ensure this gets picked up.